Microsoft.Toolkit.Uwp.UI.Controls/RangeSelector/RangeSelector.xaml

```diff
@@ -12,40 +31,33 @@
             <Setter.Value>
                 <ControlTemplate TargetType="controls:RangeSelector">
                     <Grid x:Name="ControlGrid" Height="24" >
-                        <Grid.Resources>
-                            <Style x:Key="SliderThumbStyle" TargetType="Thumb">
-                                <Setter Property="UseSystemFocusVisuals" Value="True"/>
-                                <Setter Property="BorderThickness" Value="0"/>
-                                <Setter Property="Background" Value="{ThemeResource SliderThumbBackground}"/>
-                                <Setter Property="Height" Value="24"/>
-                                <Setter Property="Width" Value="8"/>
-                                <Setter Property="FocusVisualMargin" Value="-14,-6,-14,-6"/>
-                                <Setter Property="Template">
-                                    <Setter.Value>
-                                        <ControlTemplate TargetType="Thumb">
-                                            <Border Background="{TemplateBinding Background}" 
-                                                    BorderThickness="{TemplateBinding BorderThickness}" 
-                                                    BorderBrush="{TemplateBinding BorderBrush}" 
-                                                    CornerRadius="4"/>
-                                        </ControlTemplate>
-                                    </Setter.Value>
-                                </Setter>
-                            </Style>
-                        </Grid.Resources>
+                        <Grid.ColumnDefinitions>
+                            <ColumnDefinition Width="Auto"/>
+                            <ColumnDefinition Width="12"/>
+                            <ColumnDefinition Width="*"/>
+                            <ColumnDefinition Width="12"/>
+                            <ColumnDefinition Width="Auto"/>
+                        </Grid.ColumnDefinitions>
 
-                        <Border x:Name="OutOfRangeContentContainer"                                 
-                                Background="Transparent">
+                        <TextBlock Grid.Column="0" x:Name="MinValueText" Text="Min"/>
```

Comment:
Slider doesn't have a textblock showing the current value. Why would RangeSelector have one? Remove the min and max textblocks. The Tooltip should be enough and most likely the user will have a value displayed somewhere already

Reply:
Why was this kept? ping @nmetulev